Makefile

```diff
@@ -1,33 +1,84 @@
-PROMBENCH_CMD        = /bin/prombench
+PROMBENCH_CMD        = ./prombench
 DOCKER_TAG = docker.io/sipian/prombench:v2.0.0
 
-deploy:
+create-nodepool:
+	printf ">> Creating NodePools"
 	$(PROMBENCH_CMD) gke nodepool create -a /etc/serviceaccount/service-account.json \
 		-v ZONE:${ZONE} -v PROJECT_ID:${PROJECT_ID} -v CLUSTER_NAME:${CLUSTER_NAME} -v PR_NUMBER:${PR_NUMBER} \
 		-f  components/prombench/nodepools.yaml
 
+DIR := "/go/src/github.com/${REPO_OWNER}/${REPO_NAME}"
+start-pr:
+	mkdir -p /go/src/github.com
+	printf "\n\n>> Fetching Pull Request"
+	git clone https://github.com/${REPO_OWNER}/${REPO_NAME} ${DIR}
+
+	cd ${DIR} && \
+	git fetch origin pull/${PR_NUMBER}/head:pr-branch && \
+	git checkout pr-branch && \
+	printf "\n\n>> Creating prometheus binaries" && \
+	make build && \
+	printf "\n\n>> Starting prometheus" && \
+	./prometheus --config.file=/etc/prometheus/config/prometheus.yaml \
+          		 --storage.tsdb.path=/data \
+				 --web.console.libraries=${DIR}/console_libraries \
+            	 --web.console.templates=${DIR}/consoles
+
+deploy-pr: create-nodepool
+	printf ">> Deploying Prombench components for PR"
+	$(PROMBENCH_CMD) gke resource apply -a /etc/serviceaccount/service-account.json \
+		-v ZONE:${ZONE} -v PROJECT_ID:${PROJECT_ID} -v CLUSTER_NAME:${CLUSTER_NAME} -v PR_NUMBER:${PR_NUMBER} \
+		-v PROMETHEUS_1_NAME:${PROMETHEUS_1_NAME} -v PROMETHEUS_1_IMAGE:${PROMETHEUS_1_IMAGE} \
+		-v PROMETHEUS_2_NAME:${PROMETHEUS_2_NAME} -v PROMETHEUS_2_IMAGE:${DOCKER_TAG} \
```

Comment:
It is not clear where PROMETHEUS_1_NAME and PROMETHEUS_2_NAME are set , can you add some comments somewhere.

Reply:
Prow config has the following args set in it's config
- PROJECT_ID
- ZONE
- CLUSTER_NAME

When the start-benchmark prow-job is created, the benchmark plugin add the following args
- ACTION 
- PR_NUMBER
- PROMETHEUS_1_NAME
- PROMETHEUS_1_IMAGE
- PROMETHEUS_2_NAME
- PROMETHEUS_2_IMAGE
The values of these args are not constant are are dependent on `/benchmark pr|release`